app/controllers/bulk_update/placements_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module BulkUpdate
+  class PlacementsController < ApplicationController
+    before_action { require_feature_flag(:bulk_placements) }
+
+    helper_method :bulk_placements_count, :organisation_filename
+
+    def new
+      @navigation_view = ::Funding::NavigationView.new(organisation:)
+    end
+
+  private
+
+    def organisation
+      @organisation ||= current_user.organisation
+    end
+
+    def bulk_placements_count
+      @bulk_placements_count ||= current_user.organisation.trainees.without_placements.count
+    end
+
+    def organisation_filename
```

Comment:
Leaving this here for the time being as it may be used when responding with the csv template.